onnxruntime/core/providers/webgpu/reduction/reduction_ops.cc

```diff
@@ -82,6 +82,14 @@ REGISTER_UNARY_ELEMENTWISE_VERSIONED_KERNEL(ReduceLogSumExp, 11, 12);
 REGISTER_UNARY_ELEMENTWISE_VERSIONED_KERNEL(ReduceLogSumExp, 13, 17);
 REGISTER_UNARY_ELEMENTWISE_KERNEL(ReduceLogSumExp, 18);
 
+REGISTER_UNARY_ELEMENTWISE_VERSIONED_KERNEL(ArgMax, 1, 10);
```

Comment:
https://github.com/microsoft/onnxruntime/pull/24077 renames this function, will need to rebase

Reply:
k, will do once it is merged

Reply:
done